src/assets/scss/ui/swipeableList.scss

```diff
@@ -0,0 +1,55 @@
+.swipeable-list {
+  padding: 2px;
```

Comment:
Do we need to add the styles manually, or we can import it from plugin?

Reply:
We have already imported the plugin css, but that contains basic classes
import "react-swipeable-list/dist/styles.css";